{"timestamp":"2025-05-20T06:29:28.405432","level":"info","event":"DAG bundles loaded: dags-folder","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-05-20T06:29:28.408129","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/sql-test.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-05-20T06:29:28.566748","level":"info","event":"Secrets backends loaded for worker","count":1,"backend_classes":["EnvironmentVariablesBackend"],"logger":"supervisor"}
{"timestamp":"2025-05-20T06:29:28.732911","level":"info","event":"Connection Retrieved 'mssqlserver'","logger":"airflow.hooks.base"}
{"timestamp":"2025-05-20T06:29:28.804455","level":"info","event":"Running statement: CREATE TABLE pet2 (\n        pet_id SERIAL PRIMARY KEY,\n        name VARCHAR NOT NULL,\n        pet_type VARCHAR NOT NULL,\n        birth_date DATE NOT NULL,\n        OWNER VARCHAR NOT NULL);\n        , parameters: None","logger":"airflow.task.hooks.airflow.providers.microsoft.mssql.hooks.mssql.MsSqlHook"}
{"timestamp":"2025-05-20T06:29:28.863767","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"OperationalError","exc_value":"(2715, b'Column, parameter, or variable #1: Cannot find data type SERIAL.DB-Lib error message 20018, severity 16:\\nGeneral SQL Server error: Check messages from the SQL Server\\n')","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":838,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1130,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":408,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":212,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":235,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/sql-test.py","lineno":24,"name":"extract_from_mssql"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py","lineno":729,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/common/sql/hooks/sql.py","lineno":777,"name":"_run_command"},{"filename":"/opt/airflow/src/pymssql/_pymssql.pyx","lineno":465,"name":"pymssql._pymssql.Cursor.execute"}]},{"exc_type":"MSSQLDatabaseException","exc_value":"(2715, b'Column, parameter, or variable #1: Cannot find data type SERIAL.DB-Lib error message 20018, severity 16:\\nGeneral SQL Server error: Check messages from the SQL Server\\n')","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/opt/airflow/src/pymssql/_pymssql.pyx","lineno":447,"name":"pymssql._pymssql.Cursor.execute"},{"filename":"/opt/airflow/src/pymssql/_mssql.pyx","lineno":1125,"name":"pymssql._mssql.MSSQLConnection.execute_query"},{"filename":"/opt/airflow/src/pymssql/_mssql.pyx","lineno":1156,"name":"pymssql._mssql.MSSQLConnection.execute_query"},{"filename":"/opt/airflow/src/pymssql/_mssql.pyx","lineno":1289,"name":"pymssql._mssql.MSSQLConnection.format_and_run_query"},{"filename":"/opt/airflow/src/pymssql/_mssql.pyx","lineno":1855,"name":"pymssql._mssql.check_cancel_and_raise"},{"filename":"/opt/airflow/src/pymssql/_mssql.pyx","lineno":1901,"name":"pymssql._mssql.raise_MSSQLDatabaseException"}]}]}
